app/components/article-content.js

```diff
@@ -534,6 +543,15 @@ export default Component.extend(
 		handleCollapsibleSections() {
 			toArray(this.element.querySelectorAll('h2[section]'))
 				.forEach((header) => header.addEventListener('click', this.handleCollapsibleSectionHeaderClick.bind(this)));
+		},
+
+		uncollapseSections() {
+			toArray(this.element.querySelectorAll('h2[section]'))
+				.forEach((header) => {
+					if (!header.classList.contains('open-section')) {
```

Comment:
It could be removed by adding to h2 selector `:not(.open-section)`. I'm not sure about performance, but it should be cleaner

Reply:
You right, cleaner and that should be faster too.